docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.9"
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password
+  web:
```

Comment:
since we are dual booting, we should have a way to run docker with the next rails version too

Reply:
we could do something like  `docker-compose run -e BUNDLE_GEMFILE=Gemfile.next web [COMMAND]`

Reply:
Ahh, I got what you mean, somehow I missed the guide link...